custom_components/xiaomi_miio_airpurifier/fan.py

```diff
@@ -806,7 +806,7 @@
 )
 
 SERVICE_SCHEMA_PTC_LEVEL = AIRPURIFIER_SERVICE_SCHEMA.extend(
-    {vol.Required(ATTR_LEVEL): vol.In([level.name for level in AirfreshT2017PtcLevel])}
+    {vol.Required(ATTR_PTC_LEVEL): vol.In([level.name for level in AirfreshT2017PtcLevel])}
```

Comment:
For consistency I want to stick to `level` here. If you use the `set_fan_led_brightness` service you set the  brightness` and not the `led_brightness` f.e.